<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkullAndScalpDialog</class>
 <widget class="QDialog" name="SkullAndScalpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Skull and Scalp Tool</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_12">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0" colspan="2">
     <widget class="QCheckBox" name="skullComputeThresholdsCheckBox">
      <property name="text">
       <string>Compute thresholds</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="skullThresholdLabel">
      <property name="text">
       <string>Skull threshold</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="skullThresholdEditBox">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="scalpThresholdLabel">
      <property name="text">
       <string>Scalp threshold</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="scalpThresholdEditBox">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runStepButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Step &gt;</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="previousStepButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt; Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="stepNameText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Next step: initialize</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Skull and Scalp Tool</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="bfcButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Stage: Bias Field Correction</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restoreFromPreviousSessionButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore From Previous...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
